rotkehlchen/globaldb/schema.py

```diff
@@ -315,6 +315,14 @@
 );
 """
 
+DB_CREATE_LOCATION_UNSUPPORTED_ASSETS = """
+CREATE TABLE IF NOT EXISTS location_unsupported_assets (
+    location TEXT NOT NULL,
+    exchange_symbol TEXT NOT NULL,
+    UNIQUE (location, exchange_symbol)
```

Comment:
since the locationa and symbol are not null you can do 

```
    FOREIGN KEY(location) REFERENCES location(location) ON UPDATE CASCADE,
    FOREIGN KEY(to_asset) REFERENCES assets(identifier) ON UPDATE CASCADE
```